test/quantization/core/test_quantized_op.py

```diff
@@ -2574,15 +2574,18 @@ def test_instance_norm(self):
         combined = [shape_list, torch_types, y_scales, y_zero_points, channels_last_list, affine_list]
         test_cases_product = itertools.product(*combined)
         test_cases = list(test_cases_product)
-        # add just one test case to test overflow
-        test_cases.append([
-            [1, 4, 224, 224, 160],  # shape,
-            torch.qint8,  # torch_type
-            0.1,  # scale
-            0,  # zero_point
-            False,   # channels_last
-            True,  # affine
-        ])
+        # NB: Add just one test case to test overflow, but this case is too slow to run
+        # internally in @fbcode//mode/dev, the long pole is the 4x calls to torch.sort
+        # inside torch.unique current implementation
+        if not IS_SANDCASTLE:
```

Comment:
Should we keep it with ASAN as well?

Reply:
Noted.  We could do this preemptively, although OSS CI is green there and I don’t see it shows up on https://github.com/pytorch/test-infra/blob/generated-stats/stats/slow-tests.json.  Let me double check if the test is run on ASAN and how long it takes